src/box/vinyl.c

```diff
@@ -1627,12 +1623,13 @@ vy_check_is_unique_secondary(struct vy_tx *tx, const struct vy_read_view **rv,
 {
 	assert(lsm->opts.is_unique);
 	if (!lsm->cmp_def->is_multikey) {
-		return vy_check_is_unique_secondary_one(tx, rv, lsm, stmt,
-							MULTIKEY_NONE);
+		return vy_check_is_unique_secondary_one(tx, rv,
+				lsm, stmt, MULTIKEY_NONE);
```

Comment:
Nit: Bad indentation - the continuation should be aligned by the parenthesis. I fixed checkpatch - now it detects this error.

Reply:
Fixed.